### PR TITLE
Add tests for LWC

### PR DESCRIPTION
**Pull request by ekashida**
Everything looks good in my local build but please let me know if I've missed anything.

I can open a PR against https://github.com/webcomponents/custom-elements-everywhere/pull/2263 with the required changes after this is merged.

### CODE REVIEW

File: README.md
Comment (nolanlawson) [inline README.md:34]: It's nice that we're adding this – is this the only existing component that was missed here?

File: libraries/lwc/karma.conf.js
Comment (nolanlawson) [inline libraries/lwc/karma.conf.js:87]: Should this be removed?

File: libraries/lwc/src/basic-tests.js
Comment (nolanlawson) [inline libraries/lwc/src/basic-tests.js:174]: Do we need this commented-out code?

**Review by nolanlawson — APPROVED** [review]
LGTM!

File: README.md
Comment (ekashida) [reply README.md:34]: Looks like `React@experimental` was also missed

File: libraries/lwc/src/basic-tests.js
Comment (ekashida) [reply libraries/lwc/src/basic-tests.js:174]: I implemented these before realizing that the other libraries had commented them out so I went ahead and did the same. I'll just remove it.

**Comment by ekashida** [comment]
@rictic @justinfagnani this PR is ready for review

**Comment by nolanlawson** [comment]
Friendly ping @rictic @justinfagnani – any chance of getting LWC on Custom Elements Everywhere? :bow:

**Comment by gked** [comment]
Is this getting merged? I would love to reference the LWC results that are publicly available.

File: README.md
Comment (justinfagnani) [reply README.md:34]: Nice catch. Thanks!

File: libraries/lwc/meta/issues.json
Comment (justinfagnani) [inline libraries/lwc/meta/issues.json:3]: This issue is closed now. Should it be https://github.com/salesforce/lwc/issues/3335 ?

File: libraries/lwc/meta/summary.md
Comment (justinfagnani) [inline libraries/lwc/meta/summary.md:11]: No library has restrictions on `addEventListener()` since it's a native method. Do other libraries/frameworks mention this this way?

**Review by justinfagnani — APPROVED** [review]
LGTM!

**Comment by ekashida** [comment]
Thanks for the review @justinfagnani!

The PR mentions that it has workflows awaiting approval by a maintainer. Is this something you could help us with?

**Pull request merged**
